docs/performance/tune-performance.md

```diff
@@ -148,6 +161,9 @@ Memory consumption can be reduced between multiple sessions by configuring the s
 * Inter op num threads (used only when parallel execution is enabled) is not affected by OpenMP settings and should
 always be set using the ORT APIs.
 
+### Custom threading hooks
+For customer prefers to create and manage their own threads, please see usage [here](https://github.com/microsoft/onnxruntime/blob/6284cbe833bc47eb90ed4082ff2ae015b8e5c004/onnxruntime/test/shared_lib/test_inference.cc#L1867) and [here](https://github.com/microsoft/onnxruntime/blob/6284cbe833bc47eb90ed4082ff2ae015b8e5c004/onnxruntime/test/global_thread_pools/test_main.cc#L36).
```

Comment:
Instead of referring to line numbers that can change, can you directly include the example here?

Reply:
Examples attached.